### PR TITLE
Update npm packages

### LINKED ISSUE #564: some npm packages are deprecated

### PR DESCRIPTION
**Pull request by seanRong**
This fixes #564

### CODE REVIEW

File: templates/vue/package.json [hunk 1]
```diff
@@ -19,12 +19,14 @@
     "babel-polyfill": "^6.26.0",
     "bootstrap": "^4.3.1",
     "bootstrap-vue": "^2.13.1",
+    "jquery": "^1.9.1",
```
Comment (narendrasss) [inline templates/vue/package.json:22]: Any reason why this was installed? We don't use jQuery in the vue build.

File: templates/vue/package.json [hunk 2]
```diff
@@ -19,12 +19,14 @@
...
     "moment": "^2.25.3",
     "moment-timezone": "^0.5.28",
     "vue": "^2.6.10",
     "vue-router": "^3.1.3",
     "vue-slicksort": "^1.1.3",
-    "vuex": "^3.1.1"
+    "vuex": "^3.1.1",
+    "watchpack-chokidar2": "^2.0.0"
```
Comment (narendrasss) [inline templates/vue/package.json:29]: What is this package for?

File: templates/vue/config-sample.js [hunk 3]
```diff
@@ -1 +0,0 @@
-export const LINK_PREVIEW_API_KEY = ""
```
Comment (narendrasss) [inline templates/vue/config-sample.js:1]: Please keep this file. We need it for new users so they know what the `config.js` file should look like. It's also unrelated to the deprecated problem (to my understanding). We want to keep the PRs as contained to their original issue as possible.

File: templates/vue/package.json [hunk 4]
```diff
@@ -41,7 +43,7 @@
     "cross-env": "^5.0.5",
     "css-loader": "^0.28.7",
     "cypress": "^3.8.0",
-    "eslint": "^6.5.1",
+    "eslint": "^5.16.0",
```
Comment (narendrasss) [inline templates/vue/package.json:46]: This looks like we're downgrading to me. Could you try updating this to `7.1.0` and see if that works? We have a pending PR at the moment upgrading the eslint version to that version.

File: templates/vue/package.json (hunk 2 above)
Comment (seanRong) [reply templates/vue/package.json:29]: I believe this came in when I attempted to update fsevents, which I couldn't/ didn't need to update anyway.

File: templates/vue/package.json (hunk 1 above)
Comment (seanRong) [reply templates/vue/package.json:22]: Will remove

**Comment by narendrasss** [comment]
Closing as not needed as per our discussion.

**Pull request closed without merging**